protocol-specs/generators/dpos_forger_selection/sample_generator.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { getAddressFromPassphrase } = require('@liskhq/lisk-cryptography');
+const { Mnemonic } = require('@liskhq/lisk-passphrase');
+const crypto = require('crypto');
+const fs = require('fs');
+
+const [amount = 150, filePath, fixedValue] = process.argv.slice(2);
+
+const generateDelegates = (num, fixedNum) => {
+	const delegateList = [];
+	for (let i = 0; i < num; i += 1) {
+		const passphrase = Mnemonic.generateMnemonic();
+		const address = getAddressFromPassphrase(passphrase);
+		const buf = crypto.randomBytes(8);
+		const randomNumber = buf.readBigUInt64BE() / BigInt(10) ** BigInt(8);
```

Comment:
Will, it does not generate random data on each run? I remember while having a discussion on another PR you highlighted that protocol-spec JSON should not change the output unless there is a change in protocol.

Reply:
This is just a tool I used to generate the input JSON. It is not called directly in the generator, so it will not generate random data